bn254/src/lib.rs

```diff
@@ -192,6 +205,15 @@ mod target_arch {
             )?;
             Ok(Self(pod_bytes))
         }
+
+        #[inline(always)]
+        fn from_le_bytes(le_bytes: &[u8]) -> Result<Self, AltBn128Error> {
```

Comment:
Likewise, can you update the docs on top of `pub struct PodG2(pub [u8; G2_POINT_SIZE]);`? Here, I think it is important to make clear of the ordering convention of the polynomial variables.